projects/plugins/jetpack/modules/search/instant-search/components/search-result-expanded.scss

```diff
@@ -5,10 +5,6 @@ $image-square-size: 128px;
 	display: flex;
 	flex-flow: column;
 
-	@include break-small-up() {
-		flex-flow: row nowrap;
-	}
-
```

Comment:
With this removal, I think we need to tweak the small viewport styling for expanded search results:

<table>
<tr>
	<td>Before</td>
	<td>After</td>
<tr>
	<td><img width="570" alt="Screen Shot 2021-06-22 at 2 25 10 PM" src="https://user-images.githubusercontent.com/4044428/122994466-bbbfd100-d365-11eb-8c32-e5076dbcb6ef.png"></td>
	<td><img width="570" alt="Screen Shot 2021-06-22 at 2 28 06 PM" src="https://user-images.githubusercontent.com/4044428/122994842-15c09680-d366-11eb-94ec-6bfaf9b24be4.png"></td>

</table>

Note that this issue is exacerbated by the fact that the multisite result format is being applied universally to all sites, as mentioned in a comment for `search-results.jsx`.

Reply:
Should be fixed now.